src/lsst/cmservice/config.py

```diff
@@ -25,6 +25,7 @@ class Configuration(BaseSettings):
     database_password: str | None = Field(
         title="The password for the cm-service database",
         validation_alias="CM_DATABASE_PASSWORD",
+        default=None,
```

Comment:
These keep trying to sneak back in.  I'm concerned about when/where this is needed in your work flow?

Reply:
One of those overenthusiastic context checking things, I think some safir thing that checks to config model.